_episodes/01-intro.md

```diff
@@ -8,11 +8,10 @@ objectives:
 - "Explain how the shell relates to the keyboard, the screen, the operating system, and users' programs."
 - "Explain when and why command-line interfaces should be used instead of graphical interfaces."
 keypoints:
-- "Explain the similarities and differences between a file and a directory."
-- "Translate an absolute path into a relative path and vice versa."
-- "Construct absolute and relative paths that identify specific files and directories."
 - "Explain the steps in the shell's read-run-print cycle."
```

Comment:
Maybe I'm missing something, but it seems like this first list of keypoints (up to line 18 in the new version) don't do anything, because they just get overwritten by the next array of keypoints.